modules/dynamic-plugins/con-rhdh-plugins.adoc

```diff
@@ -0,0 +1,70 @@
+[id="con-rhdh-plugins"]
+
+= Plugins in {product}
```

Comment:
I've reviewed the content in this PR: https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/547. Could you please take a look at my comments and incorporate them here?